lte/gateway/release/build-magma.sh

```diff
@@ -219,6 +235,10 @@ make build_oai BUILD_TYPE="${BUILD_TYPE}"
 make build_session_manager BUILD_TYPE="${BUILD_TYPE}"
 make build_sctpd BUILD_TYPE="${BUILD_TYPE}"
 
+# Build Envoy Controller service
+cd "${MAGMA_ROOT}/feg/gateway"
```

Comment:
This is the build script why would you need that there?

Reply:
^just followed the above make builds for other services

Reply:
Yeah but those (session_manager, sctpd) are packages we build from our custom source. not needed for envoy.